kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
 VNAME=voyager-operator  # release name of Voyager
+VVERS=v12.0.0-rc.1      # release version of Voyager
```

Comment:
Why would we recommend a release candidate?

Reply:
Because it's the latest and it works, where 11.0.1 seems to have issues.

Reply:
and I would not consider this a recommendation, it's a sample, Voyager lists what versions of Kubernetes they support...